CMSgov/design-system
# [WNMGDS-1857] MGOV typography to tokens, doc site updates

## Summary

- Moved several typography settings into theme configuration options so medicare styles will carry through to gatsby docs site
- Extended gatsby headings with `ds-hX` classes to pick up correct font sizing & family settings
- Updated mgov theme with correct font token overrides
- Updated loki snapshots (from Sarah's recent Autocomplete bugfix #2063)
- Updated several class names to use correct token names instead of older style names
- changed `build-tokens` to `build:tokens` to match our `verb:object` layout

## How to test

1. `yarn install && yarn start` - switch themes, check out typography pages, medicare and other themes
